### PR TITLE
[WINDUP-3268] - JDK11 upgrade - MTA CLI Scipt

### PR DESCRIPTION
**Pull request by carlosthe19916**

### CODE REVIEW

**Comment by PhilipCattanach** [comment]
@carlosthe19916 Hi Carlos - It is not "11 or higher" it is just "11" 
MTA does not work with say Java 17.

**Comment by carlosthe19916** [comment]
> @carlosthe19916 Hi Carlos - It is not "11 or higher" it is just "11" MTA does not work with say Java 17.

Thanks for the correction Phil! The latest commit should've fixed it

**Comment by PhilipCattanach** [comment]
@Carlos - Did you notice the Tech Preview line
https://github.com/windup/windup-distribution/pull/81/files#diff-664f0b026a48572812a832dce512927594405449e42dc0ca8d18e38d88c779c7R225
That is redundant isn't it?

**Comment by carlosthe19916** [comment]
> @carlos - Did you notice the Tech Preview line https://github.com/windup/windup-distribution/pull/81/files#diff-664f0b026a48572812a832dce512927594405449e42dc0ca8d18e38d88c779c7R225 That is redundant isn't it?

@PhilipCattanach Very good point, that line was introduced a long time ago when JDK 11 was in preview, but now JDK 11 is no longer just a TECH PREVIEW. Line removed.

File: src/main/resources/bin/mta-cli [hunk 1]
```diff
@@ -220,13 +220,11 @@ fi
 MODULES=""
 JAVAVER=`"$JAVACMD" -version 2>&1`
 case $JAVAVER in
-"11."*)
+*"11"*)
```
Comment (carlosthe19916) [inline src/main/resources/bin/mta-cli:223]: Just for adding a not about this change. It seems our master branch content is wrong and after an internal chat with @mrizzi we noticed the condition code needed to be fixed. I've tested and verified this condition locally.

**Comment by PhilipCattanach** [comment]
@carlosthe19916 The validation works perfectly for java 8 and 11. However if I try to run the cli with Java 17 it craps out with MTA execution failed due to: null
java.lang.ExceptionInInitializerError
Can you extend the validation to also capture Releases above 11?

**Comment by PhilipCattanach** [comment]
Sorry for the delay in retesting this @carlosthe19916. It now provides the error message if the using a version of Java less than or greater than 11. Which is what we want. Thanks.

**Pull request merged**